### PR TITLE
Github Actions file update

### PR DESCRIPTION
**Pull request by EdwinGuzman**
This updates the Github Actions standards file to include script examples in Digital for running unit tests and deploying to AWS. This could use more examples from other projects so please feel free to let me know so I can add them.

Open questions:
- Not sure if the "Reporting results to Slack" section is still applicable.

### CODE REVIEW

File: standards/github-actions.md [hunk 1]
```diff
@@ -1,3 +1,60 @@
...
+    role-to-assume: arn:aws:iam::[REPLACE-THIS]:role/GithubActionsDeployerRole
+    aws-region: us-east-1
+```
+
+- In the build step, make sure to update the `ECR_REPOSITORY` with the app's ERC repo name.
+- In the `aws ecs update-service` command step, update the app's ECS cluster and service name.
+
+## Tests and other actions
+
+Github Actions are also great for runnign unit tests, creating git tags, publishing to npm, or running any other automated tasks.
```
Comment (keithbauer) [inline standards/github-actions.md:46]:
```suggestion
Github Actions are also great for running unit tests, creating git tags, publishing to npm, or running any other automated tasks.
```

**Review by keithbauer — APPROVED** [review]
Clearly explains the benefits of Github Actions and provides good examples.

I'm also unsure about the `Reporting results to Slack` section since none of the projects I work with use this functionality, but perhaps others do.

File: standards/github-actions.md [hunk 2]
```diff
@@ -1,3 +1,60 @@
...
+- [research-catalog](https://github.com/NYPL/research-catalog/blob/main/.github/workflows/testing.yml) (node)
+- [digital-collections](https://github.com/NYPL/digital-collections/blob/main/.github/workflows/ci.yml) (node)
+
+Other actions include but are not limited to:
+- [deploying to Github pages](https://github.com/NYPL/nypl-design-system/blob/development/.github/workflows/gh-pages.yml)
+- [creating tags and publishing to npm](https://github.com/NYPL/nypl-design-system/blob/development/.github/workflows/release.yml)
+
 ## Reporting results to Slack
 
 We use the [action-slack-notify](https://github.com/rtCamp/action-slack-notify) Github Action to post the results of GitHub actions to Slack. Here's an example configuration from the [circulation-patron-web](https://github.com/NYPL-Simplified/circulation-patron-web/) project, which posts a link to a report from an integration test suite:
```
Comment (mwbenowitz) [inline standards/github-actions.md:60]: I'm not aware of this still working on any repos. I don't think its necessary as GHA will email people about failed jobs, so maybe we should replace this with a section on how to configure those notifications? (Unless Slack would be more useful as GH emails can be very noisy and tend to be ignored)

File: standards/github-actions.md [hunk 3]
```diff
@@ -1,17 +1,56 @@
...
+```yml
+- name: Configure AWS credentials
+  uses: aws-actions/configure-aws-credentials@v2
+  with:
+    role-to-assume: arn:aws:iam::[REPLACE-THIS]:role/GithubActionsDeployerRole
+    aws-region: us-east-1
 ```
 
-In this case, the `SLACK_WEBHOOK` secret is set to the [test report WebHook URL](ci.md).
+- In the build step, make sure to update the `ECR_REPOSITORY` with the app's ERC repo name.
```
Comment (7emansell) [inline standards/github-actions.md:41]: Nit: ECR

**Pull request merged**